setup.js

```diff
@@ -44,3 +45,13 @@ function getMongodbMemoryOptions() {
     };
   }
 }
+
+function getMongoURLEnvName() {
+  try {
+    const {mongoURLEnvName} = require(resolve(cwd, 'jest-mongodb-config.js'));
+
+    return mongoURLEnvName;
```

Comment:
this looks like even if mongoURLEnvName undefined it will be used here

Reply:
thanks, I fixed it by returning 'MONGO_URL' if mongoURLEnvName is undefined.